src/Umbraco.Web.UI.Client/src/less/belle.less

```diff
@@ -73,7 +73,8 @@
 @import "modals.less";
 @import "panel.less";
 @import "sections.less";
-@import "helveticons.less";
+//@import "helveticons.less";
```

Comment:
```suggestion
```
No need to keep commented out code, that's what Giit history is for.

Reply:
Yes, sure 🙂